src/components/clothingGrid/TitleWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from "react";
+
+type TitleWrapperProps = {
+  children: ReactNode;
+};
+
+export default function TitleWrapper({ children }: TitleWrapperProps) {
```

Comment:
The wheel is already invented :)
Moreover, try to normalize arrow function vs named function usage
```suggestion
import type { PropsWithChildren } from "react";

export const TitleWrapper = ({ children }: PropsWithChildren) => {
```